PipecatClientIOSDaily.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name         = 'PipecatClientIOSDaily'
+  s.version      = '0.3.1'
+  s.summary      = 'Daily WebRTC Transport for Pipecat client iOS.'
```

Comment:
I would say something more like "Pipecat iOS SDK with Daily WebRTC Transport", since it's the client and transport bundled together

Reply:
"Pipecat iOS SDK with Daily WebRTC Transport" is what we use in the README.

But, come to think of it...maybe it should be "Pipecat _Client_ iOS SDK with Daily WebRTC Transport"

Reply:
Actually, to be consistent with the terminology in https://github.com/pipecat-ai/pipecat-client-ios/pull/16/files, I suppose it should be "Pipecat iOS client library with Daily WebRTC transport"